Rename Services to Actions in integrations: D (#33809)
home-assistant/home-assistant.github.io: source/_integrations/downloader.markdown

ha_config_flow: true
---

> The **Downloader** {% term integration %} provides a action to download files. It will raise an error and not continue to set itself up when the download directory does not exist. The directory needs to be writable for the user that is running Home Assistant.

Author: ```suggestion
The **Downloader** {% term integration %} provides an action to download files. It will raise an error and not continue to set itself up when the download directory does not exist. The directory needs to be writable for the user who is running Home Assistant.
```
